gdk-pixbuf/Gir.toml

```diff
@@ -13,7 +13,9 @@ generate = [
     "GdkPixbuf.InterpType",
     "GdkPixbuf.PixbufAlphaMode",
     "GdkPixbuf.PixbufError",
+    "GdkPixbuf.PixbufFormatFlags",
```

Comment:
this one doesn't seem to be used by any public api, at least that's introspectable;